prime-router/settings/organizations-local.yml

```diff
@@ -47,57 +47,11 @@
     jurisdiction: STATE
     stateCode: AZ
     receivers:
```

Comment:
Removing the TEST/IN submissions to AZ from our test 'organizations-local.yml', which is only used by localhost on-laptop runs of docker.